example/lib/custom_mapping/custom_mapping.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:smartstruct/smartstruct.dart';
+
+part 'custom_mapping.mapper.g.dart';
+
+class Source {
+  final String firstName;
+  final String lastName;
+  final String birthDate;
+  final String avatar;
+
+  Source(
+    this.firstName,
+    this.lastName,
+    this.birthDate,
+    this.avatar,
+  );
+}
+
+class Target {
+  final String fullName;
+  final DateTime birthDate;
```

Comment:
Only constructor parameters are paid attention to by the generator.
If i change the Target class in the example custom_mapping.dart to the following.
```dart
class Target {
  final String fullName;
  DateTime? birthDate;
  final String avatar;

  Target(
    this.fullName,
    this.avatar,
  );
}
```
With the custom mapping of birthdate (not changed)
```dart
@Mapper()
abstract class CustomMapper {
  @DateTimeFromSourceMapper(targetField: "birthDate", f: mapBirthDate)
  @CustomMapping(targetField: "fullName", f: mapFullName)
  Target fromSource(Source source);
}
```
The birthDate is ignored by the generator.